### PR TITLE
Token file updates for testnet and devnet

### PR DESCRIPTION
**Pull request by banshee**

### CODE REVIEW

**Review by 59023g — APPROVED** [review]

**Comment by banshee** [comment]
Don't merge this without UI team approval, probably requires UI config changes as well

**Review by khdegraaf — APPROVED** [review]

**Review by 59023g — APPROVED** [review]

**Review by khdegraaf — APPROVED** [review]

**Pull request merged**
